jboss-kie-kieserver/added/launch/jboss-kie-kieserver-hooks.sh

```diff
@@ -22,7 +22,7 @@ update_config_map() {
             -H 'Accept: application/json' \
             -H 'Content-Type: application/strategic-merge-patch+json' \
             -XPATCH \
-            https://${KUBERNETES_SERVICE_HOST:-kubernetes.default.svc}:${KUBERNETES_SERVICE_PORT:-443}/api/v1/namespaces/${namespace}/configmaps/myapp-kieserver -d ${payload} )
+            https://${KUBERNETES_SERVICE_HOST:-kubernetes.default.svc}:${KUBERNETES_SERVICE_PORT:-443}/api/v1/namespaces/${namespace}/configmaps/${KIE_SERVER_ID} -d ${payload} )
```

Comment:
@spolti The KIE Server DC name is not necessarily to the same as KIE Server id due to one of the enhancements specified by this JIRA (https://issues.jboss.org/projects/JBPM/issues/JBPM-8150). However, we do mandate that all the OpenShift resources, such as DC, Service, Route and CM associated with one KIE Server uses exactly the same name, which could be a generated synthetic name like: 'myapp-kieserver-0'. Then, the actual KIE Server ID will be carried by a reserved label: 'services.server.kie.org/kie-server-id'. 
As a result, we may want to do something like this:
```
kieDCName=$(echo ${kieResponse:: -3} | python -c 'import json,sys;obj=json.load(sys.stdin);print (obj["items"][0]["metadata"]["name"])')
```
Then, pass that as extra argument to function update_config_map().